scripts/postToGitHub.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+'use strict';
+
+const octokit = require('@octokit/rest')();
+let token = process.env.GITHUB_TOKEN;
+
+if(token != undefined) {
+    octokit.authenticate({
+        type: 'token',
+        token: token
+    });
+}
+
+module.exports = {
+    postGithubComment: function(owner, repository, prNumber, commentBody) {
```

Comment:
Are we concerned that every time a commit is pushed, a new build will run and this will create a new comment? If so, maybe we can put an identifier in the comment and then either create a new comment or update the existing one if found. SwaggerToSdk is already doing this (it matches on the header text) but you should also be able to do it by inserting an HTML comment like `<!-- momentoftruth -->` which will be invisible to the user but will make it easy to locate the comment to update.